conf/script/common/README.md

```diff
@@ -0,0 +1 @@
+## README for common modules
```

Comment:
Better text?

Reply:
Added : Placeholder for common configuration files in v1 and v2.